evo-delivery-app/evo-delivery-frontend/src/features/map/drawers/ResultsDrawer.tsx

```diff
@@ -0,0 +1,42 @@
+import {useContext} from 'react';
+import {OrderMarker} from '../markers/OrderMarker';
+import {MapContext} from '../context';
+import {Depot, EvaluateResult} from "../../../../../evo-delivery-backend/src/types";
+import {DepotMarker} from "../markers/DepotMarker";
+import {TrackMarker} from "../markers/TrackMarker";
+
+
+export const ResultsDrawer = () => {
+    const {state: mapState} = useContext(MapContext);
+    const {result} = mapState;
+    const {routes, depot} = result as EvaluateResult;
+
+
+    const renderOrdersMarkers = () => (
+        routes?.flatMap((driverRoute) => (
+            driverRoute.orders.map((order) => (
+                <OrderMarker order={order} key={order._id}/>
+            ))
+        ))
+    );
+
+    const renderDepotMarker = () => (
+        <DepotMarker depot={depot as Depot} key={depot?._id}/>
+    );
```

Comment:
Maybe we should consider extracting this logic to the runs page (or however that updates the state).

I mean that the map doesn't know the connections between route, order, and depot.
We will provide 3 different entities and the map will draw them for us.

Reply:
isn't it what it already does?

Reply:
maybe i didn't understand